src/app/core/services/kafka/converters/converter.service.ts

```diff
@@ -87,13 +89,16 @@ export abstract class ConverterService {
   }
 
   getRadarSpecifications(): Promise<any[] | null> {
-    return this.http
-      .get(DefaultSchemaSpecEndpoint)
-      .toPromise()
-      .then(
-        res => (this.specifications = YAML.parse(atob(res['content'])).data)
+    return this.remoteConfig
+      .read()
+      .then(config =>
+        config.getOrDefault(
+          ConfigKeys.KAFKA_SPECIFICATION_URL,
+          DefaultSchemaSpecEndpoint
+        )
       )
-      .then(specs => (this.specifications = specs))
+      .then(url => this.http.get(url).toPromise())
+      .then(res => YAML.parse(atob(res['content'])).data)
```

Comment:
do you need to set `this.specifications` ?

Reply:
Ah yes added back.